sycl/source/detail/usm/usm_impl.cpp

```diff
@@ -59,6 +59,13 @@ void *alignedAllocHost(size_t Alignment, size_t Size, const context &Ctxt,
   PrepareNotify.scopedNotify(
       (uint16_t)xpti::trace_point_type_t::mem_alloc_begin);
 #endif
+  const auto &devices = Ctxt.get_devices();
+  if (!std::any_of(devices.begin(), devices.end(), [&](const auto &device) {
+        return device.has(sycl::aspect::usm_device_allocations);
```

Comment:
Should we check aspect::usm_host_allocations here? I think `sycl::aspect::usm_device_allocations` is a mistake.

Reply:
Yes, good catch!